extensions/mssql/src/utils.ts

```diff
@@ -35,19 +35,19 @@ export function getAppDataPath() {
  * @param filePath source notebook file name
  * @param fileExtension file type
  */
-export function getTargetFileName(filePath: string): string {
+export function getSaveableFileName(filePath: string): string {
```

Comment:
this is no longer needed, right? we can simply use what ever the template notebook's name is.

Reply:
Updated this to findNextUntitledEditorName to handle conflicts with open editor names.